### PR TITLE
Floating label in liveform and livefilter

### PR DESCRIPTION
**Pull request by aparnasj**

### CODE REVIEW

**Review by swethakundaram — APPROVED** [review]

**Comment by prasanth-reddy-wavemaker-com** [comment]
Lets wait till we release 10.6.2 before we merge this PR

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/live-form/liveform-layouts.md:14]: Lets add in brackets (Angular material style)

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/live-form/liveform-layouts.md:49]: Lets add in brackets (Angular material style)

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md
Comment (aparnasj) [reply learn/app-development/widgets/datalive/live-form/liveform-layouts.md]: Added

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md
Comment (aparnasj) [reply learn/app-development/widgets/datalive/live-form/liveform-layouts.md]: Added

File: learn/app-development/widgets/datalive/form/form-layouts.md
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/form-layouts.md:13]: Link Angular material style to this https://material.angular.io/components/form-field/examples

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/live-form/liveform-layouts.md:49]: Link Angular material style to this https://material.angular.io/components/form-field/examples

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md
Comment (aparnasj) [reply learn/app-development/widgets/datalive/live-form/liveform-layouts.md:49]: done

File: learn/app-development/widgets/datalive/form/form-layouts.md
Comment (aparnasj) [reply learn/app-development/widgets/datalive/form/form-layouts.md]: done

**Pull request merged**
